scripts/index.js

```diff
@@ -0,0 +1,105 @@
+// connect the up arrow so id of up-arrow to an on click event
+// when the id is clicked, then increase temp counter by 1
+
+// temperature variable, every time the user clickes the button, the temperature count goes up by 1
+
+
+// const appearanceHeading = document.getElementById("facts__heading");
+
+// appearanceHeading.textContent = "Qualities and Traits of a Crab";
+let temperature = 80
+
+
+const increaseTemp=()=> {
+// When clicked the temperature will rise by 1 degree
+    ++temperature
+    // console.log("i am clicking the up button")
+    // console.log(temperature)
+    document.getElementById("temperature-number").innerHTML=temperature
+    // console.log('I am the color', color)    
+}
+
+
+const decreaseTemp=()=> {
+    // When clicked the temperature will drop by 1 degree
+    --temperature
+    console.log("i am clicking the down button")
+    console.log(temperature)
+    document.getElementById("temperature-number").innerHTML=temperature
+    console.log(color)
+}
+
+// New Function will change the sky in Weather Garden
+// based on the selected sky in the drop down menu
+const setTypeOfWeather=()=> {
+    let skyValue = document.getElementById("sky").value
+    let sky = document.getElementById("selected-weather")
+
+    // if skyValue == "Sunny"
+    // set sky innerHtml to a sun
+    if (skyValue=="Sunny") {
+        sky.innerHTML="☁️ ☁️ ☁️ ☀️ ☁️ ☁️";
+    } else if (skyValue=="Cloudy") {
+        sky.innerHTML="☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️";
+    } else if (skyValue=="Rainy") {
+        sky.innerHTML="🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧🌧";
+    } else if (skyValue=="Snowy") {
+        sky.innerHTML="🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨";
+    };
+    console.log(skyValue)
+}
+setTypeOfWeather()
```

Comment:
I recommend moving this call inside the 'registerEventHandlers' function, so that it's guaranteed to run after the html is loaded.